.github/workflows/release_linux.yml

```diff
@@ -0,0 +1,70 @@
+name: LinuxRelease
+
+on:
+  push:
+    branches:
+      - rel-*
+  # test this PR for now, remove it before the merge
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        #python-version: [3.6, 3.7, 3.8, 3.9] 
+        python-version: [3.8] # test
+        architecture: ['x64']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: ${{ matrix.architecture }}
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest==5.4.3 nbval numpy
+        python -m pip install virtualenv
+        python -m virtualenv py$(python.version)
+        source py$(python.version)/bin/activate
+        python -m pip install numpy protobuf==3.11.3
+        sudo apt-get install protobuf-compiler libprotoc-dev
+
+    - name: Build manylinux Python wheels
+      env:
+        ONNX_BUILD_TESTS: 1
+        USE_MSVC_STATIC_RUNTIME: 1
+        ONNX_ML: 1
+        CMAKE_ARGS: "-DONNX_USE_LITE_PROTO=ON"
+      uses: RalfG/python-wheels-manylinux-build@v0.3.3-manylinux2010_x86_64
+      with:
+        python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38m'
+        build-requirements: 'numpy protobuf'
+        system-packages: 'cmake3 glibc-devel libstdc++-devel python-devel'
```

Comment:
No, don't install python-devel. All you need should be provided by manylinux. Don't get such changes from the system packages.

Reply:
https://github.com/pybind/pybind11/issues/1728
pybind11 needs such library (Python.h) which centOS doesn't have initially... I have tried to fix it from pybind11 side but failed. Do you have any idea about how to fix this without installing python-devel? Thank you!

Reply:
Replied below.

Reply:
Do you mean here: `It depends on which cmake version you want to use...` ?
But after some experiments, either using `find_package(PythonInterp` or `find_package(Python` cannot resolve the error as follow:
```
/root/onnx/third_party/pybind11/include/pybind11/detail/common.h:112:20: fatal error: Python.h: No such file or directory
 #include <Python.h>
          ^~~~~~~~~~         
```
I also can repro this in my local CentOS. It seems to me that centOS does not have such library and it needs to be installed from `python-devel`...

Reply:
Yes. Please check how onnxruntime do this.